src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractCollectionIntegrationTestBase.java

```diff
@@ -29,23 +29,7 @@ public final void createSimpleCollection() {
   }
 
   protected void createSimpleCollection(String collectionToCreate) {
-    given()
```

Comment:
Refactoring that I thought I'd need (to create Collection from `CollectionResourceIntegrationTest`.
Ultimately not needed but seems like refactoring makes sense for future use.